### PR TITLE
change the app code as per react native document

### PR DESCRIPTION
**Pull request by sanjayradadiya**
https://react-native-community.github.io/upgrade-helper/?from=0.59.8&to=0.61.4

Fixes #

Changes in this pull request:
-
-
-

@

### CODE REVIEW

File: android/app/build.gradle [hunk 1]
```diff
@@ -93,6 +98,27 @@ def enableSeparateBuildPerCPUArchitecture = true
...
+ */
+def jscFlavor = 'org.webkit:android-jsc:+'
+/**
+ * Whether to enable the Hermes VM.
+ *
+ * This should be set on project.ext.react and mirrored here.  If it is not set
+ * on project.ext.react, JavaScript will not be compiled to Hermes Bytecode
+ * and the benefits of using Hermes will therefore be sharply reduced.
+ */
+def enableHermes = project.ext.react.get("enableHermes", false);
```
Comment (sagararyal) [inline android/app/build.gradle]: I think we should enable this, as one of the tasks is to integrate Hermes as well. for performance.
#1111

File: android/build.gradle [hunk 2]
```diff
@@ -13,7 +13,7 @@ buildscript {
         jcenter()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.5.1'
+        classpath 'com.android.tools.build:gradle:3.4.2'
```
Comment (sagararyal) [inline android/build.gradle]: is it a downgrade?

File: ios/TreecounterApp/Info.plist [hunk 3]
```diff
@@ -66,8 +56,8 @@
 	<string>We would like to use your camera for the app.</string>
```
Comment (sagararyal) [inline ios/TreecounterApp/Info.plist]: These should be translated

File: ios/TreecounterApp/Info.plist [hunk 4]
```diff
@@ -43,17 +43,7 @@
...
-			<key>localhost</key>
-			<dict>
-				<key>NSExceptionAllowsInsecureHTTPLoads</key>
-				<true/>
-			</dict>
-		</dict>
-	</dict>
+	
 	<key>NSAppleMusicUsageDescription</key>
 	<string>We would like access to your music library for the app.</string>
```
Comment (sagararyal) [inline ios/TreecounterApp/Info.plist]: translate and add to lingo hub as well

File: ios/TreecounterApp/Info.plist [hunk 5]
```diff
@@ -66,8 +56,8 @@
 	<string>We would like to use your camera for the app.</string>
 	<key>NSLocationAlwaysUsageDescription</key>
 	<string>We need your location to get the coordinates for the tree registration.</string>
-	<key>NSLocationWhenInUseUsageDescription</key>
-	<string>We need your location to get the coordinates for the tree registration.</string>
+	
+	
 	<key>NSMicrophoneUsageDescription</key>
 	<string>We would like to use your microphone (for videos).</string>
```
Comment (sagararyal) [inline ios/TreecounterApp/Info.plist]: should be translated

File: ios/TreecounterApp/Info.plist (hunk 3 above)
Comment (norbertschuler) [reply ios/TreecounterApp/Info.plist]: The permission text resources are actually translated in https://github.com/Plant-for-the-Planet-org/treecounter-app/blob/develop/ios/de.lproj/InfoPlist.strings - I have not yet tried I we can extract these resources to LingoHub.

File: android/build.gradle (hunk 2 above)
Comment (norbertschuler) [reply android/build.gradle]: Actually, our default branch is at `com.android.tools.build:gradle:3.5.2`

File: android/app/build.gradle (hunk 1 above)
Comment (norbertschuler) [reply android/app/build.gradle]: We could test it with another issue. First let's try to do the upgrade.

**Comment by norbertschuler** [comment]
I assume this got replaced by #1673?

**Pull request closed without merging**